_redirects.yml

```
@@ -96,6 +96,16 @@
   # used in warnings printed by the Docker Engine, and in the installation script
   # at "get.docker.com"
   - /go/attack-surface/
+"/registry/spec/deprecated-schema-v1/":
+  # Details about deprecated image manifest formats. This redirect is currently
+  # used in warnings printed by the Docker Engine when pulling deprecated image
+  # formats:
+  #
+  #   - Docker Image Specification v1.0.0 (https://github.com/moby/moby/blob/v24.0.0/image/spec/v1.md)
+  #   - Docker Image manifest version 2, schema 1 (https://github.com/distribution/distribution/blob/ff2bce27319a0a0bca924820c353fae3b9046e91/docs/spec/manifest-v2-1.md)
+  #
+  # TODO(thaJeztah) We need a better page describing the image formats (and deprecated ones)
+  - /go/image-specs/
```

If this redirect is meant to handle specifically the deprecated format, maybe we should include that in the slug.
```suggestion
  - /go/deprecated-image-specs/
```

---

Yes, I'm fine with that. Let me update 👍

---

Do you think plural or singular would be better? What feels most natural to you to type?

---

Either is fine for me